README.md

```diff
@@ -163,6 +166,190 @@ Then the page should be accessible checking only: document-title, label
 Then the page should be accessible according to: best-practice and checking: aria-roles, definition-list
 ```
 
+# RSpec
+
+## Configuration
+
+1. Require rspec matchers: in `spec/spec_helper.rb`, `spec/support/env.rb` or similar.
+
+    ``` ruby
+    require 'axe/rspec'
+    ```
+
+2. Configure Browser/Webdriver
+
+See [Configure Browser/Webdriver for Cucumber][configure browser webdriver] for instructions to configure the browser or webdriver for RSpec. The configuration is shared for use in RSpec.
+
+
+## Built-In Accessibility RSpec Matchers
+
+To construct an axe accessibility RSpec check, begin with `expect(page).to be_accessible`, and append any clauses necessary.
```

Comment:
I would point out somewhere (perhaps in this section's heading) that the actual name of the matcher is `BeAccessible`.